tmail-backend/mailets/src/main/scala/com/linagora/tmail/mailets/EnforceRateLimitingPlan.scala

```diff
@@ -67,7 +85,7 @@ class EnforceRateLimitingPlan @Inject()(planRepository: RateLimitationPlanReposi
       }
 
   private def retrieveRateLimiter(id: RateLimitingPlanId): SMono[Seq[TmailPlanRateLimiter]] =
-    SMono.fromPublisher(planRepository.get(id))
+    SMono.fromPublisher(planRepositoryCache.get(id))
```

Comment:
For some configuration it could be the regular one.

We need to find a better name for this field?